pnet-idp-client-openidconnect/src/main/java/at/porscheinformatik/idp/openidconnect/PartnerNetOAuth2AuthorizationRequestResolver.java

```diff
@@ -26,6 +26,7 @@
 public class PartnerNetOAuth2AuthorizationRequestResolver implements OAuth2AuthorizationRequestResolver
 {
     public static final String ACR_PARAM = "acr";
+    public static final String PRESELECT_TENANT_PARAM = "preselect_tenant";
     private final OAuth2AuthorizationRequestResolver defaultAuthorizationRequestResolver;
 
     public static UriComponentsBuilder requestNistAuthenticationLevels(UriComponentsBuilder uri,
```

Comment:
Please add a "requestPreselectedTenant" method the same way as the requestNistAuthenticationLevels method.

Reply:
method added